internal/cleanup/cleanup.go

```diff
@@ -127,7 +129,7 @@ func (h *exportCleanupHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	timeoutCtx, cancel := context.WithTimeout(ctx, h.config.Timeout)
 	defer cancel()
 
-	count, err := h.database.DeleteFilesBefore(timeoutCtx, cutoff, h.blobstore)
+	count, err := exportdb.NewExportDB(h.database).DeleteFilesBefore(timeoutCtx, cutoff, h.blobstore)
```

Comment:
Since the package is already called `exportdb`, you could call the function `New` to avoid stutter.